sale_product_set/i18n/es.po

```diff
@@ -69,24 +76,25 @@ msgid "Last Updated on"
 msgstr "Última actualización el"
 
 #. module: sale_product_set
-#: code:addons/sale_product_set/models/product_set.py:9
 #: field:product.set,name:0
-#, python-format
 msgid "Name"
 msgstr "Nombre"
 
 #. module: sale_product_set
-#: code:addons/sale_product_set/models/product_set_line.py:15
 #: field:product.set.line,product_id:0
-#, python-format
 msgid "Product"
 msgstr "Producto"
 
 #. module: sale_product_set
-#: code:addons/sale_product_set/wizard/product_set_add.py:12
 #: model:ir.actions.act_window,name:sale_product_set.act_open_product_set_view
-#: model:ir.model,name:sale_product_set.model_product_set
 #: model:ir.ui.menu,name:sale_product_set.menu_product_set
+#, fuzzy
+msgid "Product Set"
+msgstr "Producto"
```

Comment:
I understand it's a long way to get it, I would like to avoid that Github PR becomes a translation reviews avoiding bikeshedding about things I can't understand!...